.github/workflows/easy-cp.yml

```diff
@@ -57,12 +57,27 @@ jobs:
       - name: Create PR on CP success
         if: ${{ steps.attempt-cp.conclusion == 'success' }}
         working-directory: ./flutter
+        id: create-pr
         run: |
           git push https://${{ env.GITHUB_TOKEN }}@github.com/flutteractionsbot/flutter cp-${CHANNEL}-${COMMIT_SHA}
-          gh pr create --title "[CP-${CHANNEL}]${PR_TITLE}" --body-file ../PULL_REQUEST_CP_TEMPLATE.md --base ${RELEASE_BRANCH} --label "cp: review" --repo flutter/flutter --head flutteractionsbot:cp-${CHANNEL}-${COMMIT_SHA}
+          {
+            echo 'PR_URL<<EOF'
+            gh pr create --title "[CP-${CHANNEL}]${PR_TITLE}" --body-file ../PULL_REQUEST_CP_TEMPLATE.md --base ${RELEASE_BRANCH} --label "cp: review" --repo flutter/flutter --head flutteractionsbot:cp-${CHANNEL}-${COMMIT_SHA}
+            echo EOF
+          } >> "$GITHUB_ENV"
         env:
           GITHUB_TOKEN: ${{ secrets.FLUTTERACTIONSBOT_CP_TOKEN }}
           PR_TITLE: ${{ github.event.pull_request.title }}
+      - name: Leave Comment on CP success
+        if: ${{ steps.create-pr.conclusion == 'success' }}
+        run: |
+          echo $PR_URL
+          NEW_PR_NUMBER="${PR_URL##*/}"
```

Comment:
Would it be easier to leave a comment on the original PR about the CP PR instead of the current approach where we leave a comment on the new PR?

Reply:
Yeah my understanding is that commenting on the original PR or the new PR will have the same number of steps, because we want to provide the link of the new PR so that people could find it easily.

And to get the PR link of the new PR we can only parse the output from 'gh pr create'.